sphinx_gallery/gen_gallery.py

```diff
@@ -143,28 +159,75 @@ def touch_empty_backreferences(app, what, name, obj, options, lines):
         open(examples_path, 'w').close()
 
 
-gallery_conf = {
-    'filename_pattern': re.escape(os.sep) + 'plot',
-    'examples_dirs': '../examples',
-    'gallery_dirs': 'auto_examples',
-    'mod_example_dir': os.path.join('modules', 'generated'),
-    'doc_module': (),
-    'reference_url': {},
-}
+def sumarize_failing_examples(app, exception):
+    """Collects the list of falling examples during build and prints them with the traceback
+
+    Raises ValueError if there where failing examples
+"""
+    if exception is not None:
+        return
+
+    # Under no-plot execution it failing examples play no role
+    if not app.config.sphinx_gallery_conf['plot_gallery']:
+        return
+
+    gallery_conf = app.config.sphinx_gallery_conf
+    failing_examples = set(gallery_conf['failing_examples'])
+    expected_failing_examples = set(gallery_conf['expected_failing_examples'])
+
+    still_failing_examples = failing_examples.intersection(
+        expected_failing_examples)
+    expected_fail_msg = []
+    if still_failing_examples:
+        expected_fail_msg.append("Known failing examples, still failing:")
+        for fail_example in still_failing_examples:
+            expected_fail_msg.append(fail_example + ' failed leaving traceback:\n' +
+                                     gallery_conf['failing_examples'][fail_example] + '\n')
+        print("\n".join(expected_fail_msg))
+
+    unexpected_ex_fails = failing_examples.difference(
+        expected_failing_examples)
+    fail_msgs = []
+    if unexpected_ex_fails:
+        fail_msgs.append("Unexpected failing examples:")
+        for fail_example in unexpected_ex_fails:
+            fail_msgs.append(fail_example + ' failed leaving traceback:\n' +
+                             gallery_conf['failing_examples'][fail_example] + '\n')
+
+    unexpected_ex_pass = expected_failing_examples.difference(failing_examples)
+    if unexpected_ex_pass:
+        fail_msgs.append("Examples that are not failing anymore:\n" +
```

Comment:
> Maybe add something about setting the 'expected_failing_examples' value in conf.py correctly.

You mean as part of the documentation? Because is there.